be/src/service/internal_service.cpp

```diff
@@ -808,6 +808,10 @@ void PInternalServiceImplBase<T>::get_file_schema(google::protobuf::RpcControlle
 
     std::vector<SlotDescriptor> schema;
     st = p_scanner->get_schema(&schema);
+    if (!st.ok()) {
```

Comment:
Output a more concise message in the log.

Reply:
Updated.